src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    int productivity;
```

Comment:
Your variable should be private and use setter and getter for them use.

Reply:
same for Truck and Excavator

Reply:
done